src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -20,25 +23,45 @@
     public static Student[] getSampleStudents() {
         return new Student[] {
             new Student(new Name("Alex Yeoh"), new Telegram("41ex_Yo"), new Email("alexyeoh@example.com"),
-                getTagSet("Average")),
+                getTagSet("Average", "CS2103T Tutorial", "CS2100 Lab")),
             new Student(new Name("Bernice Yu"), new Telegram("b3rnice"), new Email("berniceyu@example.com"),
-                getTagSet("Good", "Experienced")),
+                getTagSet("Good", "Experienced", "CS2103T Tutorial")),
             new Student(new Name("Charlotte Oliveiro"), new Telegram("C_Ol1ve"), new Email("charlotte@example.com"),
-                getTagSet("Struggling")),
+                getTagSet("Struggling", "CS2103T Tutorial")),
             new Student(new Name("David Li"), new Telegram("li_DAvid"), new Email("lidavid@example.com"),
-                getTagSet("Weak")),
+                getTagSet("Weak", "CS2100 Lab")),
             new Student(new Name("Irfan Ibrahim"), new Telegram("IIbr4hmm"), new Email("irfan@example.com"),
                 getTagSet("Struggling")),
             new Student(new Name("Roy Balakrishnan"), new Telegram("B_Roy"), new Email("royb@example.com"),
                 getTagSet("Average"))
         };
     }
 
+    public static ModuleClass[] getSampleModuleClasses() {
+        Student[] listOfStudents = getSampleStudents();
+        Set<UUID> studentsInCs2103TTutorial = new HashSet<>(Arrays.asList(listOfStudents[0].getUuid(),
+                listOfStudents[1].getUuid(), listOfStudents[2].getUuid()));
+        Set<UUID> studentsInCs2100Lab =
+                new HashSet<>(Arrays.asList(listOfStudents[0].getUuid(), listOfStudents[3].getUuid()));
```

Comment:
Perhaps it would be a better idea to declare the students and classes as static members and then have the two methods return a list of the static members.

Also, I think the two getter methods can be made private.